ground/src/main/java/com/google/android/ground/ui/home/mapcontainer/jobs/MapUiData.kt

```diff
@@ -13,15 +13,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.google.android.ground.ui.home.mapcontainer.cards
+package com.google.android.ground.ui.home.mapcontainer.jobs
 
 import com.google.android.ground.model.job.Job
 import com.google.android.ground.model.locationofinterest.LocationOfInterest
 
 /** Data classes used to populate the Map cards (either an Loi card, or a Suggest Loi card). */
-sealed interface MapCardUiData {
+sealed interface MapUiData {
```

Comment:
Can we update the naming of this interface, members, and ktdoc to reflect the new model?

Perhaps something along the lines of `DataCollectionEntryPointData` with `SelectedLoiSheetData` and `AdHocDataCollectionButtonData`?

Reply:
Good call! Done.